cores/esp32/esp32-hal-log.c

```diff
@@ -0,0 +1,42 @@
+#ifndef __MY_LOG__
+#define __MY_LOG__
+#include "stdio.h"
+#include "esp32-hal-log.h"
+#ifdef USE_ESP32_LOG
```

Comment:
you can remove the `#ifdef` switch. Function will not be included in the final binary if not called within the application and will at the same time allow `USE_ESP32_LOG` to be defined on per file basis.

Reply:
Done! :)